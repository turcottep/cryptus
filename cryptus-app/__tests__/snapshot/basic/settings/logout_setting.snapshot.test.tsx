import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import LogoutSetting from "../../../../components/basic/settings/logout_setting/logout_setting";

describe("<LogoutSetting />", () => {
  it("should display properly", async () => {
    const tree = renderer.create(<LogoutSetting />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
