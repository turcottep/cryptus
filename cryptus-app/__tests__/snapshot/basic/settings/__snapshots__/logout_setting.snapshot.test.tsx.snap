// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogoutSetting /> should display properly 1`] = `
<div>
  <div
    onClick={[Function]}
  >
    Log out
    <div>
      <img
        src="/icons/logout_icon.png"
      />
    </div>
  </div>
</div>
`;
